scheduler/src/cook/kubernetes/api.clj

```diff
@@ -971,6 +978,7 @@
                                                              (get-in sidecar [:resource-requirements fieldname])
                                                              0))
               total-memory-request (add-as-decimals computed-mem (get-resource-requirements-fn :memory-request))
+              ; TODO: wrap in config var
```

Comment:
revert this. we actually always want limit for init container and sidecar

Reply:
done